app/controllers/items_controller.rb

```diff
@@ -25,17 +24,26 @@ def destroy
         
         if @item.destroy
             flash[:notice] = "Item was deleted."
-            redirect_to user_path(current_user)
+            #redirect_to user_path(current_user)
+            respond_to do |format|
+                #format.html
+                format.js
+            end
         else
```

Comment:
I realized after the fact, that you can do the same respond_to block in the else (if the item WASN'T deleted). Make sense?